src/main/java/uk/gov/crowncommercial/dts/scale/cat/utils/CSVFileUtils.java

```diff
@@ -0,0 +1,62 @@
+package uk.gov.crowncommercial.dts.scale.cat.utils;
+
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.io.InputStream;
+import java.io.InputStreamReader;
+import java.util.ArrayList;
+import java.util.List;
+
+import org.apache.commons.csv.CSVFormat;
+import org.apache.commons.csv.CSVParser;
+import org.apache.commons.csv.CSVRecord;
+import org.springframework.web.multipart.MultipartFile;
+
+import uk.gov.crowncommercial.dts.scale.cat.model.jaggaer.RfxTemplateMapping;
+
+public class CSVFileUtils {
+	  public static String TYPE = "text/csv";
+	  static String[] HEADERs = { "rfx_reference_code", "rfx_short_description", "commercial_agreement_number", "lot_number" };
+
+	  public static boolean hasCSVFormat(MultipartFile file) {
+
+	    if (!TYPE.equals(file.getContentType())) {
+	      return false;
+	    }
+
+	    return true;
+	  }
+	  
+	  public static List<RfxTemplateMapping> csvToRfxTemplateMappings(InputStream is) {
```

Comment:
As this method takes an InputStream as a paramater it seems reasonable to me that it can throw IOExceptions. I would therefor not catch IOExceptions/rethrow as Runtime but simply let this mthod throw IOExceptions and let the exceptions propagate up.